var/spack/repos/builtin/packages/mpfi/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mpfi(AutotoolsPackage):
+    """Library for arbitrary precision interval arithmetic based on MPFR"""
+
+    # Notice: no simple way to deduct URL from version
+    homepage = "https://perso.ens-lyon.fr/nathalie.revol/software.html"
+    url = "https://gforge.inria.fr/frs/download.php/file/37331/mpfi-1.5.3.tar.bz2"
```

Comment:
Instead of setting a package-level URL, I would just add this to 1.5.3 since it will change once a new version is added